pkg/aws/awscluster/mutate_awscluster.go

```diff
@@ -431,6 +443,46 @@ func (m *Mutator) MutateRegion(awsCluster infrastructurev1alpha2.AWSCluster) ([]
 	return result, nil
 }
 
+//MutateAnnotationNodeTerminateUnhealthy migrate NodeTerminateUnhealthy annotations from alpha to stable in case it is configured.
+func (m *Mutator) MutateAnnotationNodeTerminateUnhealthy(awsCluster infrastructurev1alpha2.AWSCluster) ([]mutator.PatchOperation, error) {
+	var result []mutator.PatchOperation
+
+	release, err := semver.New(key.Release(&awsCluster))
+	if err != nil {
+		return nil, microerror.Maskf(notAllowedError, fmt.Sprintf("AWSCluster release has invalid value '%s'.", key.Release(&awsCluster)))
+	}
+
+	// new annotation is available from release > 15.x.x
+	release15 := semver.MustParse("14.99.99")
```

Comment:
when testing `15.0.0` was considered higher than `15.0.0-calvix` which kinda break my testing so using this instead